README.md

```diff
@@ -144,7 +144,7 @@ that will behave almost exactly like the real robot.
 
 ### Install from binary packages
 1. [Install ROS2](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html). This
-      branch supports only ROS2 Humble and Rolling. For other ROS2 versions, please see the respective
+      branch supports only ROS2 Rolling. For other ROS2 versions, please see the respective
```

Comment:
```suggestion
      branch supports only ROS2 Iron and Rolling. For other ROS2 versions, please see the respective
```

Yes, we do plan to have a separate Iron branch that will be pointing to the same head as long as possible, which is why I also would be OK with stating that this branch only supports Rolling.